### DISCUSSION TITLE
TeX/

### DISCUSSION THREAD

**Discussion by giscus** [szcf-weiya/techNotes#58] [Announcements]
# TeX/

技术博客

https://tech.hohoweiya.xyz/TeX/

**Comment by szcf-weiya** [comment]
搬运下 @ZexiCAI 的发现，

> `\global\setlength\theorempreskipamount{12pt plus 5pt minus 3pt}`
这个plus minus
为啥不直接14pt
有什么特殊作用吗
http://www.cs.cmu.edu/afs/cs/academic/class/15499-s09/www/scribe/template.tex 比如说这个
但是我用两个minus就会报错
而且12pt plus 5pt minus 3pt和12pt plus 15pt minus 13pt
结果不一样
> 
> 找到了！https://en.wikibooks.org/wiki/LaTeX/Lengths
A stretching value is a length preceded by plus or minus to specify to what extent tex is authorized to change the length.
